src/Components/3DV/SceneView.tsx

```diff
@@ -1653,6 +1660,205 @@ function SceneView(props: ISceneViewProps, ref) {
         };
     }, [outlinedMeshitems, meshMap.current]);
 
+    // Handle gizmoElementItem
+    useEffect(() => {
+        debugLog(
+            'debug',
+            'adding gizmo to element based on gizmoElementItem prop' +
+                (scene ? ' with scene' : ' no scene')
+        );
+
+        if (scene && !isLoading) {
```

Comment:
could this effect be run only if gizmoElementItem is defined?

Reply:
done